src/Phinx/Console/Command/AbstractCommand.php

```diff
@@ -242,6 +242,9 @@ protected function loadConfig(InputInterface $input, OutputInterface $output)
         }
 
         $output->writeln('<info>using config parser</info> ' . $parser);
+
+        $output->writeln('<info>using migration base class</info> ' . $config->getMigrationBaseClassName(false));
```

Comment:
This could be useful for the `create` command, but I don't think it's necessary for every command.